app/demo-geant-integration/demo-geant-integration.cc

```diff
@@ -69,7 +69,15 @@ void run(std::string const& macro_filename)
     run_manager->SetUserInitialization(new FTFP_BERT{/* verbosity = */ 0});
     run_manager->SetUserInitialization(new demo_geant::ActionInitialization());
 
-    demo_geant::GlobalSetup::Instance()->SetIgnoreProcesses({"CoulombScat"});
+    std::vector<std::string> ignore_processes = {"CoulombScat"};
+    if (G4VERSION_NUMBER >= 1110)
+    {
+        CELER_LOG(warning) << "Default Rayleigh scattering 'MinKinEnergyPrim' "
+                              "is not compatible between Celeritas and "
+                              "Geant4@11.1: disabling Rayleigh scattering";
+        ignore_processes.push_back("Rayl");
```

Comment:
Why ignoring Rayleigh in demo as we now introduce a tweak to set the MinKinEnergyPrim to the Geant411.0 value (100keV)?  Do we need both (i.e., ignoring the Rayleigh process and reverting the MinKinEneryPrim for 11.1+)?

Reply:
The tweak only works if *we* control the physics list, but we're forced to accept the defaults when loading `G4EmStandardPhysics` as part of `FTFP_BERT`.

Reply:
Between the physics list construction and the run manager initialization (in which physics tables are built), the default value can be overwritten through `G4ProcessManager` (setting necessary parameters to an individual model associated with a process).  Anyway, okay for now, but we definitely need the better handle for the equal grid spacing requirement (probably supporting multiple lambda tables per model or creating physics data directly from Geant4 cross sections).

Reply:
Aha, that's cool. Yes, medium to long term we definitely need more generic ways of handling cross sections: especially of course once we start including any sort of hadronic reactions.